### PR TITLE
[DO-NOT-MERGE][test-hadoop3.2][test-maven] Investigate JAVA_HOME not being set properly

### PR DESCRIPTION
**Pull request by HyukjinKwon**
## What changes were proposed in this pull request?

Do not merge

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by wangyum** [comment]
retest this please

File: dev/run-tests-jenkins.py [hunk 1]
```diff
@@ -183,6 +183,8 @@ def main():
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop2.7"
     if "test-hadoop3.2" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop3.2"
+        os.environ["JAVA_HOME"] = "/usr/java/jdk-11.0.1"
+        os.environ["PATH"] = "/usr/java/jdk-11.0.1/bin:%s" % os.environ["PATH"]
```
Comment (wangyum) [inline dev/run-tests-jenkins.py]: Yes, we need set `PATH` for jdk 11.

Comment (HyukjinKwon) [reply dev/run-tests-jenkins.py]: Let's see if it passes and port this change into your PR #25443 and #25423

Comment (wangyum) [reply dev/run-tests-jenkins.py]: It works:
image

Comment (HyukjinKwon) [reply dev/run-tests-jenkins.py]: Nice

**Pull request closed without merging**